pypeerassets/provider/mintr.py

```diff
@@ -8,16 +10,18 @@ class Mintr(Provider):
     it only implements queries relevant to peerassets.
     This wrapper does some tweaks to output to match original RPC response.'''
 
-    def __init__(self, network="peercoin"):
+    def __init__(self):
 
-        self.net = self._netname(network)['long']
-        self.api_session = requests.Session()
+        self.net = self._netname("peercoin")['long']
```

Comment:
I'd throw exception if it's any network but Peercoin (mainnet) as that's only thing supported by Mintr now.